### PR TITLE
obo:BFO -> bfo:BFO obo:IAO -> iao:IAO

### PR DESCRIPTION
**Pull request by andrecastro0o**

### CODE REVIEW

**Comment by andrecastro0o** [comment]
@StroemPhi  wait for me to add a test, before you merge

**Comment by andrecastro0o** [comment]
Changes:
*  prefix obo: was replace with the prefix of the corresponding obo foundry ontologies:  bfo, iao, ro, obi, ico, ncbitaxon, gaz, cro
* test `def test_obo_ontos_resolution` was introduced to test IRI obo foundry ontologies
* IAO ontology IRI was unpinned  from a specific version

File: aeon.ttl [hunk 1]
```diff
@@ -1,8 +1,14 @@
 @prefix : <https://github.com/tibonto/aeon#> .
 @prefix dc: <http://dublincore.org/specifications/dublin-core/dcmi-terms/2012-06-14/> .
 @prefix ns: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
-@prefix bfo: <http://purl.obolibrary.org/obo/bfo/2020/bfo.owl#> .
-@prefix iao: <http://purl.obolibrary.org/obo/iao/2020-06-10/iao.owl#> .
```
Comment (andrecastro0o) [inline aeon.ttl:5]: This is where IAO was pinned to a specif version @StroemPhi

**Review by andrecastro0o — COMMENTED** [review]
:+1:

File: aeon.ttl (hunk 1 above)
Comment (andrecastro0o) [reply aeon.ttl:5]: Hence bfo prefix will become
`@prefix bfo: <https://standards.iso.org/iso-iec/21838/-2/ed-1/en/owl/bfo-2020.owl> .`

**Review by StroemPhi — APPROVED** [review]

**Review by andrecastro0o — COMMENTED** [review]
:+1:

**Review by andrecastro0o — COMMENTED** [review]
:+1:

**Pull request merged**